src/init.ts

```diff
@@ -4,146 +4,235 @@ import fs from 'fs';
 import inquirer from 'inquirer';
 import os from 'os';
 import path from 'path';
-import { MCP_SERVER_CONFIG, MCP_SERVER_CONFIG_FILE } from './utils/types.js';
+import {
+  MCP_SERVER_CONFIG_FILE,
+  MCP_SERVER_REMOTE_CONFIG,
+  MCP_SERVER_STDIO_CONFIG,
+} from './utils/types.js';
+
+const applicationKeyValues = {
+  cursor: 'Cursor',
+  vscode: 'VS Code',
+  'vscode-insiders': 'VS Code Insiders',
+};
+
+const applicationConfigMap = {
+  'Claude Code': {
+    filePath: path.join(os.homedir(), '.claude.json'),
+    configKey: 'mcpServers',
+  },
+  'Claude for Desktop': {
+    filePath: path.join(
+      os.homedir(),
+      'Library',
+      'Application Support',
+      'Claude',
+      'claude_desktop_config.json',
+    ),
+    configKey: 'mcpServers',
+  },
+  Cursor: {
+    filePath: path.join(os.homedir(), '.cursor', 'mcp.json'),
+    configKey: 'mcpServers',
+  },
+  'VS Code': {
+    filePath: path.join(
+      os.homedir(),
+      'Library',
+      'Application Support',
+      'Code',
+      'User',
+      'mcp.json',
+    ),
+    configKey: 'servers',
+  },
+  'VS Code Insiders': {
+    filePath: path.join(
+      os.homedir(),
+      'Library',
+      'Application Support',
+      'Code - Insiders',
+      'User',
+      'mcp.json',
+    ),
+    configKey: 'servers',
+  },
+};
 
-const supportedApplications = ['Claude for Desktop', 'Cursor AI'];
 const supportedModes = ['npx', 'docker'];
 const tigrisMcpServerImage = 'quay.io/tigrisdata/tigris-mcp-server:latest';
+const supportedTransports = ['remote', 'stdio'];
```

Comment:
Why should a user know or care about the MCP transport? I'm just trying to understand the usecase.